### PR TITLE
seminars: submissions

### PR DESCRIPTION
**Pull request by harunurhan**

### CODE REVIEW

File: backend/inspirehep/submissions/views.py [hunk 1]
```diff
@@ -158,6 +166,75 @@ def create_ticket(self, record, rt_template):
...
+            "keywords",
+            "public_notes",
+        ]
+
+        record_data = dict(record)
+        for field in deletable_fields:
+            if field not in data and field in record_data:
+                del record_data[field]
+        record_data.update(data)
+        builder = SeminarBuilder(record=record_data)
```
Comment (drjova) [inline backend/inspirehep/submissions/views.py:224]: do we neeed the builder here? Can we use the serializer?

File: e2e/tests/submissions/seminar.test.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+          email: 'contact@example.com',
+          name: 'Contact',
+        },
+      ],
+      field_of_interest: ['Accelerators', 'Math and Math Physics'],
+      series_name: 'A seminar serie',
+      series_number: '1',
+      additional_info: 'A public note',
+    });
+
```
Comment (drjova) [inline e2e/tests/submissions/seminar.test.js:59]: too many lines?

File: ui/src/submissions/index.jsx [hunk 3]
```diff
@@ -61,6 +64,16 @@ class Submissions extends Component {
               path={SUBMISSIONS_CONFERENCE}
               component={ConferenceSubmissionPageContainer}
             />
+            <Route
```
Comment (drjova) [inline ui/src/submissions/index.jsx:67]: should we put it under feature flag or only superusers/curators?

File: ui/src/submissions/seminars/components/SeminarForm.jsx [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+}
+
+function getSuggestionSourceLegacyICN(suggestion) {
+  return suggestion._source.legacy_ICN;
+}
+
+// TODO: and AM/PM display for some locales (like US)
+// TODO: series suggester
+
+const SHOW_TIME = {
```
Comment (drjova) [inline ui/src/submissions/seminars/components/SeminarForm.jsx:36]:
```suggestion
const DISPLAY_TIME = {
```

File: ui/src/submissions/seminars/components/SeminarForm.jsx [hunk 5]
```diff
@@ -0,0 +1,255 @@
...
+      </AntForm.Item>
+      <Field
+        name="field_of_interest"
+        label="* Field of Interest"
+        mode="multiple"
+        options={inspireCategoryOptions}
+        component={SelectField}
+      />
+      <ArrayOf
+        // TODO: is it worth to move this and its schema to common since it's used in jobs
```
Comment (drjova) [inline ui/src/submissions/seminars/components/SeminarForm.jsx:192]: Yes 👍

File: ui/src/submissions/index.jsx (hunk 3 above)
Comment (harunurhan) [reply ui/src/submissions/index.jsx:67]: well we don't have it under `submissions` menu and on homepage, so it's not really discoverable.

File: backend/inspirehep/submissions/views.py (hunk 1 above)
Comment (harunurhan) [reply backend/inspirehep/submissions/views.py:224]: leftover! btw we really need to check other views and simplify them similar to Seminars

File: e2e/tests/submissions/seminar.test.js [hunk 6]
```diff
@@ -0,0 +1,152 @@
...
+      ],
+      end_datetime: '2020-05-06T16:30:00.000000',
+      start_datetime: '2020-05-06T10:30:00.000000',
+      timezone: 'Europe/Zurich',
+    };
+    const { metadata } = await seminarResponse.json();
+
+    expect(metadata).toMatchObject(expectedMetadata);
+
+    await page.goto(`${routes.SEMINAR_SUBMISSION}/${seminarId}`);
```
Comment (drjova) [inline e2e/tests/submissions/seminar.test.js:135]: don't have strong opinion, but do we have it like that on the other submissions as well? Why we don't separate create and update cases?

Comment (harunurhan) [reply e2e/tests/submissions/seminar.test.js:135]: it's more complicated, since you need some initial setup to have the record in the DB.

**Pull request merged**